xdmod_data/_validator.py

```diff
@@ -255,8 +255,8 @@ def __get_dates_from_duration(duration):
     last_quarter = today + timedelta(days=-90)
     this_month_start = date(today.year, today.month, 1)
     if today.month == 1:
-        last_full_month_start_year = today.year - 1
-        last_full_month_start_month = 12
+        last_full_month_start_year = today.year - 1  # pragma: no cover
+        last_full_month_start_month = 12  # pragma: no cover
```

Comment:
I think instead of using `pragma: no cover` on these two statements, we can just put it once on the `if today.month == 1`.